objective.py

```diff
@@ -57,7 +57,7 @@ def compute(self, B):
         for Di in D:
             nll -= np.linalg.slogdet(Di)[1]
         nll /= 2 * n_matrices
-        obj = np.exp(nll)  # take exp for stopping criterion to work better
+        obj = max(nll, 0)  # can be < 0 because of numerical errors
```

Comment:
I suspect it slows the benchopt run. Can you check ?

Reply:
It does not appear to change the runtime. However, just using `obj = nll` makes the benchmark take a super long time...

Reply:
I saw the same. I suggest your revert to np.exp(nll) for now

Reply:
Done, but I think that taking the nll as objective (and fix numerical errors leading to it being < 0) is more natural.

Reply:
I don't think it's the problem here. The number of runs dependents on relative or absolute improvements and in any case use the exp is not the same.

Reply:
But why the exp and not `nll + 1`  (both fix the slowness problem)? This seems arbitrary and hackish to me. `max(nll, 0)` on the other hand is easier to understand since it just bypasses numerical errors.